packages/ui/src/date-picker/DatePicker.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { datePickerBaseStyles } from './DatepickerBase.styles';
+import { makeStyles } from '@material-ui/styles';
+import { Color } from '../theme/Color';
+
+export const datePickerStyles = {
+  ...datePickerBaseStyles,
+  selected: {
+    background: Color.Blue,
+    color: '#FFF',
+    borderRadius: '4px',
+  },
+};
+
+export const useStyles = makeStyles(datePickerStyles);
```

Comment:
I suggest renaming to `useDatePickerStyles`. We don't want accidental imports from app